modules/mobile/docs/openapi.yaml

```diff
@@ -756,7 +756,136 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Errors"
-
+  /v0/payment-information:
```

Comment:
I believe @steve-gov said there's another system that stores direct-deposit in addition to PPIU. So maybe this could be `/v0/payment-information/benefits`?

Reply:
sounds good to me, I'll change it real quick